### PR TITLE
Updating quickstart app

### PR DESCRIPTION
**Pull request by vishwajitshep0815**
## Summary
Updating Quickstart app with latest react-native coding standards and TypeScript support

### CODE REVIEW

File: reader-sdk-react-native-quickstart/index.js [hunk 1]
```diff
@@ -1,21 +1,9 @@
-/*
-Copyright 2018 Square Inc.
+/**
+ * @format
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/index.js:2]: missed header update.

File: reader-sdk-react-native-quickstart/ios/RNReaderSDKSample/Info.plist [hunk 2]
```diff
@@ -35,28 +35,14 @@
 			</dict>
 		</dict>
 	</dict>
-	<key>NSBluetoothAlwaysUsageDescription</key>
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/ios/RNReaderSDKSample/Info.plist:38]: Don't we need these value to be set any more?

Comment (hukid) [reply reader-sdk-react-native-quickstart/ios/RNReaderSDKSample/Info.plist:38]: I think we still need them: https://developer.squareup.com/docs/reader-sdk/build-on-ios#update-your-infoplist

File: reader-sdk-react-native-quickstart/ios/RNReaderSDKSampleTests/Info.plist [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/ios/RNReaderSDKSampleTests/Info.plist:1]: Do we really need this test project?

File: reader-sdk-react-native-quickstart/package.json [hunk 4]
```diff
@@ -1,47 +1,56 @@
 {
   "name": "reader-sdk-react-native-quickstart",
-  "version": "1.1.0",
+  "version": "0.0.1",
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/package.json:3]: It should be 2.0.0

**Review by hukid — CHANGES_REQUESTED** [review]

File: reader-sdk-react-native-quickstart/.bundle/config [hunk 5]
```diff
@@ -0,0 +1,2 @@
+BUNDLE_PATH: "vendor/bundle"
```
[the remark this answers is not in the document]
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/.bundle/config]: Removed it.

File: reader-sdk-react-native-quickstart/__tests__/App-test.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
```
[the remark this answers is not in the document]
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/__tests__/App-test.tsx]: Removed the  tests

File: reader-sdk-react-native-quickstart/android/app/src/main/AndroidManifest.xml [hunk 7]
```diff
@@ -1,21 +1,16 @@
...
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-
-
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools"
```
[the remark this answers is not in the document]
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/android/app/src/main/AndroidManifest.xml]: Corrected it.

File: reader-sdk-react-native-quickstart/app/components/CustomButton.tsx [hunk 8]
```diff
@@ -14,39 +14,36 @@ See the License for the specific language governing permissions and
...
-      {props.title}
-    </Text>
-  </TouchableOpacity>
-);
-
-CustomButton.defaultProps = {
-  disabled: false,
-  primary: false,
+class Props {
+  title: string = '';
```
[the remark this answers is not in the document]
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/app/components/CustomButton.tsx]: Removed default values

File: reader-sdk-react-native-quickstart/app/screens/AuthorizingScreen.tsx [hunk 9]
```diff
@@ -13,39 +13,41 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+// USEEFFCT METHOD FOR GET AUTHORIZE CODE
+  useEffect(() => {
+    authorize();
+  });
 
-  async authorize(navigation) {
-    const authCode = navigation.getParam('authCode', '');
+// AUTHORIZE METHOD
+  const authorize = async () => {
+    console.log(route.params.authCode)
```
[the remark this answers is not in the document]
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/app/screens/AuthorizingScreen.tsx]: removed it

File: reader-sdk-react-native-quickstart/index.js (hunk 1 above)
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/index.js]: Corrected the header

File: reader-sdk-react-native-quickstart/ios/RNReaderSDKSample/Info.plist (hunk 2 above)
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/ios/RNReaderSDKSample/Info.plist:38]: Added them.

File: reader-sdk-react-native-quickstart/package.json (hunk 4 above)
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/package.json]: Incremented the version

**Review by hukid — APPROVED** [review]
There are still two comments not addressed.

File: reader-sdk-react-native-quickstart/ios/RNReaderSDKSampleTests/Info.plist (hunk 3 above)
Comment (vishwajitshep0815) [reply reader-sdk-react-native-quickstart/ios/RNReaderSDKSampleTests/Info.plist]: Removed it

**Pull request merged**